packages/SystemsThinkingWorkbench-Core.package/STWSimulation.class/instance/doubleClick..st

```diff
@@ -0,0 +1,6 @@
+interaction
+doubleClick: evt
+	"create a new node"
+
+	self addNodeAt: evt cursorPoint.
+	self firstClicked: nil.
```

Comment:
why i should do this [super ...]? since we dont do it on other related stuff like mouseDown etc I dont think we should start here. doesnt gives advantages. The comment in the parent method is: **This default implementation does nothing.** I see no reason to use the super function for its functionality